FriendsWave/social/models.py

```diff
@@ -1,3 +1,90 @@
+from functools import _Descriptor
 from django.db import models
+from django.db.models.base import Model
+from django.db.models.fields.related import OneToOneField
+
+class Profil(models.Models):
+    pseudo = models.CharField(max_length=100)
+    image = models.ImageField()
+    user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
+    friends = models.ManyToManyField('self', through='Friend')
+    topics = models.ManyToManyField('Topic', through='UserTopic')
```

Comment:
topics relates to what? topics created by the user, read by the users, deleted by the user.... Use topics_of_interest: it more meaningfull